packages/sources/src/components/image/image.scss

```diff
@@ -0,0 +1,22 @@
+.img-responsive {
+  @include img-responsive;
```

Comment:
Mixins roepen we met `()` aan

Reply:
En daarnaast: Geen styling in `sources` genereren, hebben we net over de telefoon besproken 🙄